website/docs/migrating-from-docker/using-podman-mac-helper.md

```diff
@@ -25,34 +25,57 @@ The service redirects `/var/run/docker` to the fixed user-assigned UNIX socket l
 
 #### Procedure
 
-- Set up the `podman-mac-helper` service for each user.
-  Run the command:
+1. Set up the `podman-mac-helper` service: run the command in a terminal:
 
-  ```sh
-  sudo podman-mac-helper install
-  ```
-
-  For additional install options please run the command:
+   ```shell-session
+   sudo podman-mac-helper install
+   ```
 
-  ```sh
-  sudo podman-mac-helper install --help
-  ```
+1. Restart your Podman machine: go to **<icon icon="fa-solid fa-cog" size="lg" /> Settings > Resources**, and in the Podman tile, click <icon icon="fa-solid fa-repeat" size="lg" />.
 
 #### Verification
 
+1. The Docker socket is a symbolic link for the Podman socket:
+
+   ```shell-session
+   $ ls -la /var/run/docker.sock
+   ```
+
+   The output points to a `podman.sock` file such as:
+
+   ```shell-session
+   /var/run/docker.sock -> /Users/username/.local/share/containers/podman/machine/podman.sock
+   ```
+
+1. When you query the Docker socket, you receive replies from Podman rather than Docker.
+
+   For instance, this command outputs Podman version rather that Docker version:
+
+   ```shell-session
+   $ curl -s --unix-socket /var/run/docker.sock "http://v1.41/info"  | jq -r .ServerVersion
+   ```
+
 1. Your tools communicating to the Docker socket, such as [Maven](https://maven.apache.org/) or [Testcontainers](https://www.testcontainers.org/), communicate with Podman without reconfiguration.
 
-2. Use the `podman-mac-helper` tool to run commands.
-   To run a command with Podman by using the `podman-mac-helper` tool, prefix the command with `podman-mac-helper`.
+1. (Optionally, if the `docker` CLI is installed) The `docker` CLI communicates with the Podman socket.
 
-   Example:
+   Therefore this command outputs Podman version rather that Docker version:
 
    ```shell-session
-   $ podman-mac-helper run -it <your_container> bash
+   $ DOCKER_HOST=unix:///var/run/docker.sock docker info --format=json | jq -r .ServerVersion
```

Comment:
maybe remove ` DOCKER_HOST=unix:///var/run/docker.sock ` as if context is correctly set it means that docker is using this host already